common-bom/pom.xml

```diff
@@ -0,0 +1,612 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>uk.gov.justice</groupId>
+        <artifactId>maven-common</artifactId>
+        <version>1.6.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>common-bom</artifactId>
+    <packaging>pom</packaging>
+    <name>Common BOM</name>
+    <description>Common bill of materials defining standard versions for dependencies</description>
+
+    <!-- ===== NOTES ===== The dependencies defined here are NOT included in
+        a project - it just sets the version that will be included if required. You
+        still need to add the dependency in you project POM but do not define the
+        version, that it defined here so changing in one place changes in all projects
+        which ensures we dont end up using multiple versions of the same library, -->
+
+    <!-- ========== PROPERTIES ========== -->
+
+    <properties>
+        <weld-ee-version>1.0.0.CR8</weld-ee-version>
```

Comment:
Is it worth trimming this set down to only libs that we'd expect to see included in projects?